src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,17 @@
+package duke;
```

Comment:
I definitely think that more system.out.println statements can be placed in here!